### ISSUE TITLE
Bounty on qualitative analysis of interviews?

### ISSUE THREAD

**Issue by pet3r-pan** [status-im/ETHPrize#18]
I was wondering if it were possible to get a bounty on the qualitative analysis of the interview data.

Qualitative analysis include mind mapping out answers around answers to questions (eg. ) or even creating an excel sheet with quotes attached to certain topics. 

As a former ux designer, I will treat it the same way I conducted ux research and will be facilitating the effort of this this analysis with a couple other people (5 or 6 people). We are planning to do this anyways but appreciate maybe a small bounty to cover dinner or lunch. Cheers :)

**Comment by mkosowsk** [comment]
Sounds pretty compelling to me! @andytudhope what do you think? 🤔

**Comment by pet3r-pan** [comment]
In-progress https://docs.google.com/spreadsheets/d/1oj9VDHmhxPqQGKUAsISEnejdTnd-D5VAoRnVv2hDn6g/edit?usp=sharing

**Comment by andytudhope** [comment]
Yeah, of course @pet3r-pan - please make sure you log into openbounty.status.im and update your payment details there. Then, you can submit the final doc in a PR to the same dir where the initial interview spreadsheet is and uses the syntax `Fixes #18` and it will claim the bounty I will deploy here now.

Thanks for doing this, you're awesome 👍

**Comment by status-open-bounty** [comment]
![Contract deploying](https://openbounty.status.im/img/deploying_contract.png)
Transaction: [0x996e4a78e1bba771c8c6e446962aebe41f0b3bfa7c0bd1afcbe392ffddbdaff2](https://etherscan.io/tx/0x996e4a78e1bba771c8c6e446962aebe41f0b3bfa7c0bd1afcbe392ffddbdaff2)

**Comment by pet3r-pan** [comment]
Update: Summarised about 20% of the survey so far, I organised everything to a point which should be more digestible for others to dive in and take a look.

**Summarised:**
- Education: A mix bag of things to do with education, communication, development best practices etc https://docs.google.com/document/d/1kHBXS_M1BlwnVz0yTFQz2qrCMrVbsqRW0r_Li3uoL-0/edit?usp=sharing

**Yet to be summarised (but organised enough):** 
- Security
- Testing
- Tools
- More developer experience stuff
https://docs.google.com/spreadsheets/d/1oj9VDHmhxPqQGKUAsISEnejdTnd-D5VAoRnVv2hDn6g/edit?usp=sharing

However through what I absorbed, there is a lot of information which can be distilled to better design the development of popular tools and a lot of data on solidity which might be really useful to the development of say.. vyper

**Comment by andytudhope** [comment]
For further reference @pet3r-pan https://discuss.status.im/t/datasets-by-status/76

**Comment by pet3r-pan** [comment]
@mkosowsk @andytudhope First thing to come out of this https://emerald.ws/the-current-state-of-ethereum-education/

I reckon this will be an ongoing thing as more interviews are released and when I go through more of the interview.

+ Also would love to help contribute to the ETH Prize report if needed

**Comment by mkosowsk** [comment]
@pet3r-pan excellent work, I will aim to process by EOD :)

**Comment by pablanopete** [comment]
Just checking did this ever get taken care of out of band? Or period. @pet3r-pan @andytudhope

**Comment by pet3r-pan** [comment]
@pablanopete I forgot about this! Progress of this actually lead into currently ongoing work around designer education/onboarding into web3. Talked about it at the ux unconf

**Comment by andytudhope** [comment]
Yeah, afaik @pet3r-pan you did not submit anything, hence why you received no reward here? Presentations don't count, only pull requests 😉

**Comment by pet3r-pan** [comment]
@andytudhope  😢 https://github.com/status-im/ETHPrize-interviews/pull/19

**Comment by pablanopete** [comment]
@pet3r-pan @andytudhope so looks like theres' a reward here to be had? + revoking once confirmed.

**Comment by andytudhope** [comment]
I can't access the site linked because of an invalid security cert? Sorry about that, it's my oversight here.

**Comment by pet3r-pan** [comment]
@andytudhope 

Accidentally wiped my tunnel access keys to my vps and can't be arsed to fix the SSL cert for now.

Instead, I threw the article into markdown.
